src/test/gho-manager.test.ts

```diff
@@ -0,0 +1,112 @@
+import hre from 'hardhat';
+import { expect } from 'chai';
+import { makeSuite, TestEnv } from './helpers/make-suite';
+import { impersonateAccountHardhat } from '../helpers/misc-utils';
+import { ghoReserveConfig } from '../helpers/config';
+import { ONE_ADDRESS, ZERO_ADDRESS } from '../helpers/constants';
+import { ProtocolErrors } from '@aave/core-v3';
+import { getPoolConfiguratorProxy } from '@aave/deploy-v3';
+
+export const TWO_ADDRESS = '0x0000000000000000000000000000000000000002';
+
+makeSuite('Gho Manager End-To-End', (testEnv: TestEnv) => {
+  let ethers;
+
+  let poolSigner;
+  let randomSigner;
+  let poolConfigurator;
+
+  const mockLockPeriod = 3;
+
+  before(async () => {
+    ethers = hre.ethers;
+
+    const { pool } = testEnv;
+
+    poolSigner = await impersonateAccountHardhat(pool.address);
+    randomSigner = await impersonateAccountHardhat(ONE_ADDRESS);
+    poolConfigurator = await getPoolConfiguratorProxy();
+  });
+
```

Comment:
you can also add a check to ensure GhoManager is PoolAdmin

Reply:
good one `7041999aa44238888840cdc49bf17980fdcc3e8d`